package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@commercetools-frontend/ui-kit",
-  "version": "2.0.0-rc.11",
+  "version": "2.0.0",
```

Comment:
@emmenko Should we do another RC release first? Or should we publish this one and then fix any bugs in a follow-up? As we're the only consumers of ui-kit so far I guess it's fine to just go with 2.0.0 and do a bug fix release afterwards.

Reply:
I would release `3.0.0`.

> As we're the only consumers of ui-kit so far I guess it's fine to just go with 2.0.0 and do a bug fix release afterwards.

That's not true, there are already other people using this.

Reply:
Actually I would release `3.0.0-beta.0` (last time with app-kit we had a bit of post-release problems)

Reply:
Ok, let's do an RC first then 👍

Reply:
I discovered that we actually never released a `1.0.0` even - and no `2.0.0` either. And our `latest` tag points to an RC 🤕

This is quite confusing and not what I'm used to at all. 

Our published versions are 

```
[ '1.0.0-beta.0',
  '1.0.0-beta.1',
  '1.0.0-beta.2',
  '1.0.0-beta.3',
  '1.0.0-beta.4',
  '1.0.0-beta.5',
  '1.0.0-beta.6',
  '1.0.0-beta.7',
  '1.0.0-beta.8',
  '1.0.0-beta.9',
  '1.0.0-beta.10',
  '1.0.0-beta.11',
  '1.0.0-beta.12',
  '1.0.0-beta.13',
  '1.0.0-beta.14',
  '1.0.0-beta.15',
  '1.0.0-beta.17',
  '1.0.0-beta.23',
  '1.0.0-beta.24',
  '1.0.0-beta.25',
  '1.0.0-beta.26',
  '1.0.0-beta.27',
  '1.0.0-beta.28',
  '1.0.0-beta.29',
  '1.0.0-beta.30',
  '1.0.0-rollup-bundle',
  '2.0.0-rc.0',
  '2.0.0-rc.1',
  '2.0.0-rc.2',
  '2.0.0-rc.3',
  '2.0.0-rc.4',
  '2.0.0-rc.5',
  '2.0.0-rc.6',
  '2.0.0-rc.7',
  '2.0.0-rc.8',
  '2.0.0-rc.9',
  '2.0.0-rc.10',
  '2.0.0-rc.11' ]
```

I spoke to @emmenko and we agreed on releasing `3.0.0` now and to follow semver conventions  from then on (no more RCs on `latest`!).